packages/web/src/services/audius-backend/getLibs.ts

```diff
@@ -0,0 +1,11 @@
+import { FeatureFlags, RemoteConfigInstance } from '@audius/common'
+
+export const getLibs = async (remoteConfig: RemoteConfigInstance) => {
+  await remoteConfig.waitForRemoteConfig()
+  const isProxyWormholeEnabled = remoteConfig.getFeatureEnabled(
+    FeatureFlags.PROXY_WORMHOLE
+  )
+  return isProxyWormholeEnabled
```

Comment:
Could this slow down page load because we are waiting for a feature flag to start initializing libs?

Reply:
since our read flow uses eager-load i think that is okay, just write flow will take slightly longer to initialize. our sdk currently waits for remote-config too, btw